index.js

```diff
@@ -504,6 +580,13 @@ const ContainerService = {
         userContextId = args.userContextId;
```

Comment:
you must add at line 500:

if (!("source" in args)) {
  return Promise.reject(...